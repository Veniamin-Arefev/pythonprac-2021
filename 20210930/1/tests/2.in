90, 95, 33, 81, 62, 34, 2, 64, 88, 46