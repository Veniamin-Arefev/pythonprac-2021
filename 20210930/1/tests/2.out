[90, 2, 46, 95, 62, 88, 34, 81, 33, 64]
